atomic_reactor/util.py

```diff
@@ -608,6 +610,15 @@ def get_orchestrator_platforms(workflow):
             return plugin['args']['platforms']
 
 
+def get_platforms(workflow):
+    koji_platforms = workflow.prebuild_results.get(PLUGIN_CHECK_AND_SET_PLATFORMS_KEY)
+    if koji_platforms:
+        return koji_platforms
+
+    # Not an orchestrator build
+    return None
```

Comment:
If it's not an orchestrator build, how about we return a one item list with the current platform?

Reply:
I'm inclined to say no - in that in an orchestrator build the *same* thing isn't being done with the list of platforms as is done for the single platform in a worker build. [That would argue that utils.get_orchestrator_platforms() should be a private utility function and this function should be called get_orchestrator_platforms()]

Examining the 5 call sites, 4 wouldn't care one way or the other - the 5th would need some code changes:

**Only in the orchestrated case**:  plugins/build_orchestrate_build.py, plugins/post_pulp_pull.py, plugins/pre_resolve_composes.py
**Does nothing if the platform list is None *or* has a single element**: plugins/pre_pull_base_image.py
**Requires changes**:  plugins/pre_add_filesystem.py

add_filesystem does: 

        self.architectures = get_platforms(self.workflow)
        self.is_orchestrator = True if self.architectures else False

Reply:
Ok, good point about `add_filesystem`.